### PR TITLE
Improved Portuguese translation

### PR DESCRIPTION
**Pull request by maiconkkl**

### CODE REVIEW

**Comment by maiconkkl** [comment]
adjusted

File: locales/br.lua [hunk 1]
```diff
@@ -6,50 +6,50 @@ Locales['br'] = {
   ['give'] = 'dar',
   ['remove'] = 'remover',
   ['return'] = 'voltar',
-  ['give_to'] = 'give to',
+  ['give_to'] = 'dar para',
   ['amount'] = 'quantidade',
   ['giveammo'] = 'Dar muniçao',
```
Comment (ElPumpo) [inline locales/br.lua]: fix

Comment (maiconkkl) [reply locales/br.lua]: fixed

**Comment by ElPumpo** [comment]
Thanks for aiding the cause

**Pull request merged**